bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -253,7 +253,7 @@
       <div class="container">
         <div class="all-download">
           <p class="message">{{ _('Download the latest version of Firefox') }}</p>
-          <a rel="external" href="{{ firefox_url('desktop', 'all') }}">{{ _('All Firefox downloads') }}</a>
+          <a rel="external" href="{{ firefox_url('desktop', 'all', channel=channel_name|lower()) }}">{{ _('All Firefox downloads') }}</a>
```

Comment:
It looks like `lower()` is needed here, without it the page throws a `NoReverseMatch`